<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="createListsButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Create lists</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="list1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="list2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxToSecond">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>to second</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxToFirst">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>to first</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Add:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>325</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
